an-der-uni/urz.tex

```diff
@@ -39,8 +39,8 @@ \subsection*{Drucken}
 Das zum Drucken benötigte Gut\-ha\-ben wird über die CampusCard abgerechnet, die man in den Mensen und in der \gls{UB} aufladen kann. Für größere Aufträge gibt es einen Druckerraum im Keller des \gls{URZ}, der von der Firma Ricoh betrieben wird.
 Darüber hinaus bietet das URZ einen Poster-Druckservice\footnote{\url{http://www.urz.uni-heidelberg.de/service-katalog/druckservice/poster-service}} und einen 3D-Druckservice\footnote{\url{http://www.urz.uni-heidelberg.de/service-katalog/druckservice/3d-druckservice.html}} an.
 
-\subsection*{CIP-Pools -- oder -- Internet ohne Notebook}
-Rechnerräume stehen z.\,B.\ im \gls{KIP}, im \gls{PI}, im 1.~OG, im Keller der Institutsbibliothek der angewandten Mathe (\Gls{INF} 294), im \Gls{OMZ} und natürlich im \gls{URZ} bereit. In letzterem sogar mehrere -- wo noch Plätze frei sind erfahrt ihr am Infobildschirm (linker Eingang).
+\subsection*{CIP-Pools -- oder -- Internet ohne Notebook} %TODO, OMZ gibts nicht mehr, 294 ist auch falsch
+%Rechnerräume stehen z.\,B.\ im \gls{KIP}, im \gls{PI}, im 1.~OG, im Keller der Institutsbibliothek der angewandten Mathe (\Gls{INF} 294), im \Gls{OMZ} und natürlich im \gls{URZ} bereit. In letzterem sogar mehrere -- wo noch Plätze frei sind erfahrt ihr am Infobildschirm (linker Eingang).
```

Comment:
das muss aber wieder eingefügt werden und ist nur auskommentiert, weil das noch jemand überarbeiten muss, das Ersti-Info aber in der obigen Form nicht bauen würde. Ist also nicht das gleiche Argument.